.github/workflows/ci.yml

```diff
@@ -58,7 +59,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 14.x
+          node-version: 20.x
```

Comment:
got these running against node 20 which is not required, but gives us a bit more coverage in terns of testing
node 18+ has issues for the node tests in the non ember-try run which seem to be test infrastructure related only and would not affect users